### PR TITLE
Added notification messages for create/update/etc helpers

### PR DESCRIPTION
**Pull request by schlupov**

### CODE REVIEW

File: airgun/entities/activationkey.py [hunk 1]
```diff
@@ -32,23 +36,29 @@ def read(self, entity_name):
 
     def update(self, entity_name, values):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.flash.assert_no_error()
```
Comment (abalakh) [inline airgun/entities/activationkey.py]: you should assert and dismiss alerts after update, not before, as there won't be any notifications before update :)

File: airgun/entities/contentcredential.py [hunk 2]
```diff
@@ -31,6 +35,8 @@ def read(self, entity_name):
 
     def update(self, entity_name, values):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.flash.assert_no_error()
```
Comment (abalakh) [inline airgun/entities/contentcredential.py]: same here

File: airgun/entities/contentview.py [hunk 3]
```diff
@@ -35,11 +39,15 @@ def read(self, entity_name):
 
     def update(self, entity_name, values):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.flash.assert_no_error()
```
Comment (abalakh) [inline airgun/entities/contentview.py]: >same here

File: airgun/entities/hostcollection.py [hunk 4]
```diff
@@ -31,11 +35,15 @@ def read(self, entity_name):
 
     def update(self, entity_name, values):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.flash.assert_no_error()
```
Comment (abalakh) [inline airgun/entities/hostcollection.py]: same here and for the rest of `update` methods

File: airgun/entities/template.py [hunk 5]
```diff
@@ -28,16 +30,22 @@ def clone(self, entity_name, values):
...
         view.search(entity_name)
         view.table.row(name=entity_name)['Actions'].widget.fill('Lock')
+        view.flash.assert_no_error()
+        view.flash.dismiss()
 
     def unlock(self, entity_name):
         view = self.navigate_to(self, 'All')
         view.search(entity_name)
         view.table.row(name=entity_name)['Actions'].widget.fill('Unlock')
+        view.flash.assert_no_error()
```
Comment (abalakh) [inline airgun/entities/template.py]: if there's browser alert - it should be handled before any other interaction with the page (like dismissing notifications). Please move `self.browser.handle_alert()` before these lines

**Comment by oshtaier** [comment]
Looks good
Changes affected almost helpers, so full automation re-run will be good. Maybe, you can put run on a night

**Comment by schlupov** [comment]
@oshtaier I sent an e-mail with a link to the job on Jenkins. Is it ok?

**Comment by abalakh** [comment]
Haven't received any email. @schlupov @oshtaier what're the results? Imo we're pretty safe to merge this PR right now when all comments addressed, as both `flash. assert_no_error()` and `flash.dismiss()` are asserting/dismissing alerts only if they're present and do nothing if they aren't there.

**Comment by oshtaier** [comment]
I received email, but it was not using proper airgun branch, only robottelo one.
I just wanted to make sure that we do not add messages in situation when they are not expected at all, like when they should not be displayed

**Comment by oshtaier** [comment]
okay, I will try each entity manually and if all of them contains flash messages, I will merge the PR

**Comment by schlupov** [comment]
@oshtaier I sent an e-mail with link: https://satellite-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Satellite%206/job/satellite6-standalone-automation/1300/testReport/
In console output is: Collecting git+https://github.com/schlupov/airgun.git@flash (from -r requirements.txt (line 30)) so it takes proper airgun branch but I can't fill in field ROBOTTELO_REPO airgun branch, I needed to make a PR into robottelo with changed requirements :-(

File: airgun/entities/lifecycleenvironment.py [hunk 6]
```diff
@@ -15,12 +15,16 @@ def create_environment_path(self, values):
         view = self.navigate_to(self, 'New Path')
         view.fill(values)
         view.submit.click()
+        view.flash.assert_no_error()
```
Comment (oshtaier) [inline airgun/entities/lifecycleenvironment.py]: same here

File: airgun/entities/lifecycleenvironment.py [hunk 7]
```diff
@@ -15,12 +15,16 @@ def create_environment_path(self, values):
...
+        view.flash.dismiss()
 
     def create_environment(self, values, entity_name):
         view = self.navigate_to(
             self, 'New Environment', entity_name=entity_name)
         view.fill(values)
         view.submit.click()
+        view.flash.assert_no_error()
```
Comment (oshtaier) [inline airgun/entities/lifecycleenvironment.py]: same here

File: airgun/entities/product.py [hunk 8]
```diff
@@ -17,12 +17,16 @@ def create(self, values):
         view = self.navigate_to(self, 'New')
         view.fill(values)
         view.submit.click()
+        view.flash.assert_no_error()
```
Comment (oshtaier) [inline airgun/entities/product.py]: same here

**Pull request merged**